controllers/src/test/scala/cromwell/pipeline/controller/ProjectConfigurationControllerTest.scala

```diff
@@ -67,5 +67,29 @@ class ProjectConfigurationControllerTest extends AsyncWordSpec with Matchers wit
         }
       }
     }
+
+    "deactivate configuration" should {
+
+      "return success for deactivation configuration" in {
+        when(configurationService.deactivateConfiguration(configuration.projectId))
+          .thenReturn(Future.successful("Success"))
+        Delete("/configurations?project_id=" + configuration.projectId.value) ~> configurationController.route(
```

Comment:
Maybe it is easier to use concatenation here, but maybe, to hold one solid style you should rather change it to interpolation?

Reply:
It's controversial, look at the UserControllerTest and ProjectControllerTest, we used the same style there. I think this is a good direction for a new task ;)